cognite/powerops/resync/v2/README.md

```diff
@@ -0,0 +1,143 @@
+
+```
+generator:
+  production_min:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].p_min"
+    is_list: False
+    default_value: 0.0
+    cast_type: float
+```
+
+A general configuration for all instances of a specific type can be defined as above in the "data model configuration"
+file but to override a configuration on a specific instance you can use the below syntax in the "data model population"
+files.
+
+Below is an instance of a plant in the `plant_water_value_based.yaml` and the *production_min* property is being
+overridden by a custom `extraction_path`. This option currently only supports providing a `source_file` and the
+`extraction_path` in the following syntax
+
+`"[SOURCE:<source_file>]<extraction_path>"`
+
+```
+- name: Lund
+  display_name: Lund
+  ordering: 1
+  asset_type: plant
+  production_min: "[SOURCE:files/model.yaml]model.plant.Lund.p_min"
+```
+
+### References
+
+Since there are many fields that references other fields or existing CDF resources you can use their external_id or
+name to reference them using the below syntax.
+
+When using `[external_id]` as the prefix like in *market_configuration* below then the following string must be an
+exact match to an external_id. This can refer to any type in CDF as long as the full external_id is provided.
+
+When using `[name]` as the prefix like in *price_area* below then the following string must be an exact match to the
+name of an instance of the expected type for that field as the expected external_id will be extrapolated from the
+inferred type and name. For example, the expected type is price_area so the generated external_id would be
+`price_area_no2`
+
+When using `[name|type:DataModelType]` as the prefix like the *partials* below, then it will handle it the same as
+`[name]` except instead of inferring the type it will use the provided type in order to generate the external_id.
+For example, the two *partials* external_ids would become `water_value_based_partial_bid_configuration_plant_lund` and
+`shop_based_partial_bid_configuration_plant_lund_set_a`
+
+```
+- name: Mixed Configuration
+  market_configuration: "[external_id]market_configuration_nord_pool_day_ahead"
+  price_area: "[name]NO2"
+  bid_date_specification: '[["shift", {"days": -400}], ["floor", "day"]]'
+  partials:
+    - "[name|type:WaterValueBasedPartialBidConfiguration]Plant Lund"
+    - "[name|type:ShopBasedPartialBidConfigurationWrite]Plant Lund Set A"
+```
+
+## Usage
+
+See available commands:
+
+```bash
+$ powerops --help
+```
+
+Example of showing plan changes:
+
+```bash
+$ powerops plan2 resync_v1/resync_configuration.yaml
+```
+
+Example of showing apply changes:
+
+```bash
+$ powerops apply2 resync_v1/resync_configuration.yaml
+```
+
+[!NOTE]
+Current implementation requires suffixing every command with "2" in order to use the FDM v1 compatible resources.
```

Comment:
Can we switch from `plan2` and `apply2` to plan1 and apply1? (I understand that this is "the second way of doing it", hence `2`, but I feel it may be less confusing if we use `1` for `v1`...)